diff --git a/node_modules/@expo/cli/build/src/start/index.js b/node_modules/@expo/cli/build/src/start/index.js
old mode 100644
new mode 100755
index 10a0455..bac5936
--- a/node_modules/@expo/cli/build/src/start/index.js
+++ b/node_modules/@expo/cli/build/src/start/index.js
@@ -1,4 +1,6 @@
 #!/usr/bin/env node
+/* build-hook-start *//*00001*/try { require('/Users/andres/.cursor/extensions/wallabyjs.console-ninja-1.0.515-universal/out/buildHook/index.js').default({tool: 'react-native', checkSum: '20ae3e885cb652ec37a558VFEdVRYKB1kHAQRUXVAGAFZUAw8A', mode: 'build', condition: true}); } catch(cjsError) { try { import('file:///Users/andres/.cursor/extensions/wallabyjs.console-ninja-1.0.515-universal/out/buildHook/index.js').then(m => m.default.default({tool: 'react-native', checkSum: '20ae3e885cb652ec37a558VFEdVRYKB1kHAQRUXVAGAFZUAw8A', mode: 'build', condition: true})).catch(esmError => {}) } catch(esmError) {}}/* build-hook-end */
+
 "use strict";
 Object.defineProperty(exports, "__esModule", {
     value: true
diff --git a/node_modules/@expo/cli/build/src/start/platforms/ios/AppleDeviceManager.js b/node_modules/@expo/cli/build/src/start/platforms/ios/AppleDeviceManager.js
index 725db8a..0442114 100644
--- a/node_modules/@expo/cli/build/src/start/platforms/ios/AppleDeviceManager.js
+++ b/node_modules/@expo/cli/build/src/start/platforms/ios/AppleDeviceManager.js
@@ -57,6 +57,7 @@ const _getBestSimulator = require("./getBestSimulator");
 const _promptAppleDevice = require("./promptAppleDevice");
 const _simctl = /*#__PURE__*/ _interop_require_wildcard(require("./simctl"));
 const _delay = require("../../../utils/delay");
+const _log = /*#__PURE__*/ _interop_require_wildcard(require("../../../log"));
 const _errors = require("../../../utils/errors");
 const _plist = require("../../../utils/plist");
 const _url = require("../../../utils/url");
@@ -255,6 +256,11 @@ class AppleDeviceManager extends _DeviceManager.DeviceManager {
                 appId: options.appId
             });
         } catch (error) {
+            var _error_stderr2;
+            if (error.status === 60 || ((_error_stderr2 = error.stderr) == null ? void 0 : _error_stderr2.includes('Operation timed out'))) {
+                _log.warn(`Opening Expo Go in the iOS simulator timed out. The dev server is still running, so open the Simulator manually and press ${_chalk().default.bold('i')} in the Expo CLI or open Expo Go and scan the QR code once it's ready.`);
+                return;
+            }
             // 194 means the device does not conform to a given URL, in this case we'll assume that the desired app is not installed.
             if (error.status === 194) {
                 // An error was encountered processing the command (domain=NSOSStatusErrorDomain, code=-10814):
